scripts/prepare_compliance_data.py

```diff
@@ -0,0 +1,155 @@
+"""
+A script that takes the compliance dataset (the released version
+of which is at https://github.com/ga4gh/compliance/tree/master/test-data)
```

Comment:
it could default to pull from get, but needs to be parameterized,
since it might come from a fork or locally modified data.,

Danny Colligan <notifications@github.com> writes:
> In scripts/prepare_compliance_data.py:
> 
> > @@ -0,0 +1,155 @@
> > +"""
> > +A script that takes the compliance dataset (the released version
> > +of which is at https://github.com/ga4gh/compliance/tree/master/test-data)
> 
> If this is always in a well known place, why doesn't the script download it?
> 
> —
> Reply to this email directly or view it on GitHub.*
>